backend/src/db/seed.sql

```diff
@@ -0,0 +1,70 @@
+DROP TABLE IF EXISTS users, characters, avatars, charStats, equipment, charactersEquipment, monsters;
+
+
+
+CREATE TABLE users (
+    user_key SERIAL PRIMARY KEY,
+    username varchar(50),
+    email VARCHAR(80),
+    password VARCHAR(100),
+);
+
+CREATE TABLE characters (
+    character_key SERIAL PRIMARY KEY,
+    char_name VARCHAR(80),
+    [description] VARCHAR(255),
+    user_key INT REFERENCES users(user_key),
+    avater_key INT REFERENCES avatars(avatar_key),
+    gold INT,
+    experience INT,
+    [level] INT
+);
+
+CREATE TABLE avatars (
+    avatar_key SERIAL PRIMARY KEY,
+    [filename] VARCHAR(100),
```

Comment:
just a note, if we end up hosting pre-made avatars (or custom, for that matter) in AWS S3, we might want to have a column here for that URL

Reply:
S3 gives you a filename/key that you can use.  So we could just leave it the way it is currently set up. Using the key is a little bit harder to do because you have to make the request from the server vs the URL which you could just use wherever.  

Personally my vote is for the URL (I don't think there is any major difference in our application).